model/TriplesManagerService.php

```diff
@@ -190,4 +202,13 @@ protected function isFileReference(Triple $triple)
         }
     }
 
+    private function addFilePrefixForAssets(Triple $triple): void
```

Comment:
Not big deal, but If possible, move the private methods to after public/protected as we agreed in our conventions.